README.md

```diff
@@ -33,9 +33,10 @@ All images for the `octoprint/octoprint` image are multi-arch images, and we pub
     - [Configuration](#configuration)
       - [Enabling Webcam Support with Docker](#enabling-webcam-support-with-docker)
       - [Container Environment based configs](#container-environment-based-configs)
+      - [Restarting OctoPrint](#restarting-octoprint)
       - [Editing Config files manually](#editing-config-files-manually)
   - [Without docker-compose](#without-docker-compose)
-  - [Building the image yourself](#building-the-image-yourself)
+  - [Building the image yourself](docs/README.md#building-your-own-octoprint-image)
```

Comment:
Can you change the link text to `Building your own OctoPrint Image`?

Reply:
Derp, now that would make sense, wouldn't it! Fixed. Sorry for the delay...